lib/elixir/lib/float.ex

```diff
@@ -401,26 +402,34 @@ defmodule Float do
   """
   @doc since: "1.4.0"
   @spec ratio(float) :: {pos_integer | neg_integer, pos_integer}
```

Comment:
Spec should be corrected since `0` can be returned:

`@spec ratio(float) :: {integer, pos_integer}`

Reply:
Thanks!